core/variant/variant_parser.cpp

```diff
@@ -865,13 +865,47 @@ Error VariantParser::parse_value(Token &token, Variant &value, Stream *p_stream,
 			}
 
 			get_token(p_stream, token, line, r_err_str);
-			if (token.type != TK_NUMBER) {
+			// Permit empty RID.
+			if (token.type == TK_PARENTHESIS_CLOSE) {
+				value = RID();
+				return OK;
+			} else if (token.type != TK_NUMBER) {
 				r_err_str = "Expected number as argument";
 				return ERR_PARSE_ERROR;
 			}
 
 			value = token.value;
```

Comment:
Currently this *doesn't* load an `RID`, but an integer, unsure how to deal with that